### PR TITLE
DI-221 Add demo postman collection files

### PR DESCRIPTION
**Pull request by mik3ola**
## Link to JIRA Ticket

https://nhsd-jira.digital.nhs.uk/browse/DI-221

## Description

A postman collection containing a series of demo or simulated change event data modified to satisfy the following scenarios:
- The Pharmacy matches to a number of DoS services and the website is different
- The Pharmacy matches to a number of DoS Services but there is no change of website i.e. no CRs raised
- The Pharmacy doesn't match into DoS i.e. there are no services with that ODSCode. 
- The NHS.UK Changed event is not for a Pharmacy i.e. the organisation type is not PHA (e.g Dentist)

NB. Stages from the above scenarios are logged on AWS Cloudwatch and on Splunk logs 

### Noteworthy Changes

- Additional demo folder containing postman collection file
- Multiple demo change event files

## Testing

Please tick the testing that has been completed

- [ ] Manual test

## Developer Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have run the [code formatting checks](../README.md#code-quality)
- [ ] I have run the [code quality checks](../README.md#code-quality)
- [ ] New code meets [standards](https://nhsd-confluence.digital.nhs.uk/display/DI/DI+Ways+of+Working) agreed by the 

## Code Reviewer Checklist

- [ ] I can confirm the changes have been tested or approved by a tester

### CODE REVIEW

**Review by rohith1122 — APPROVED** [review]
Looks good to me apart from one comment about specified opening times

**Review by mabe13 — APPROVED** [review]

File: test/demo/demo_events/demo_event5_openingtimes_changes.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  "County": null,
+  "Latitude": 53.38030624389648,
+  "Longitude": -1.4826949834823608,
+  "Postcode": "NG5 2JJ",
+  "Phone": "0115 9606272",
+  "Email": "health.my-pharmacy@nhs.net",
+  "Website": null,
+  "OrganisationSubType": null,
+  "OrganisationAliases": [],
+  "OpeningTimes": [{
```
[the remark this answers is not in the document]
Comment (mik3ola) [reply test/demo/demo_events/demo_event5_openingtimes_changes.json:25]: Understood. Will do for subsequent scenarios. This scenario however only covers test for Standard openingtimes. Specified opening times not considered in this scenario.

**Review by mabe13 — DISMISSED** [review]

File: test/demo/collection/demo_postman_collection.json [hunk 2]
```diff
@@ -0,0 +1 @@
...,\r\n    \"Latitude\": 53.38030624389648,\r\n    \"Longitude\": -1.4826949834823608,\r\n    \"Postcode\": \"NG5 2JJ\",\r\n    \"Phone\": \"0115 9606272\",\r\n    \"Email\": \"health.my-pharmacy@nhs.net\",\r\n    \"Website\": null,\r\n    \"OrganisationSubType\": null,\r\n    \"OrganisationAliases\": [],\r\n    \"OpeningTimes\": [{\r\n      \"Weekday\": \"Monday\",\r\n      \"Times\": \"09:15-12:45\",\r\n      \"OffsetOpeningTime\": 540,\r\n      \"OffsetClosingTime\": 780,\r\n      \"OpeningTimeType\": \"General\",\r\n      \"AdditionalOpeningDate\": \"\",\r\n      \"IsOpen\": true\r\n    },\r\n    {\r\n      \"Weekday\": \"Monday\",\r\n      \"Times\": \"14:00-17:30\",\r\n      \"OffsetOpeningTime\": 840,\r\n      \"OffsetClosingTime\": 1050,\r\n      \"OpeningTimeType\": \"General\",\r\n      \"AdditionalOpeningDate\": \"\",\r\n      \"IsOpen\": true\r\n    }]\r\n}","options":{"raw":{"language":"json"}}},"url":{"raw":"https:\/\/uec-dos-integration-di-192.k8s-nonprod.texasplatform.uk\/api\/v1\/nhsuk-event-receiver\/test","protocol":"https","host":["uec-dos-integration-di-192","k8s-nonprod","texasplatform","uk"],"path":["api","v1","nhsuk-event-receiver","test"]}},"response":[]}]}]}
```
Comment (ed-sparkes) [inline test/demo/collection/demo_postman_collection.json:1]: Can this be formatted so its readable?

Also if you havent already please can you add the x-correlation-id header to each of the api requests, should be different for each of the scenarios

**Review by ed-sparkes — APPROVED** [review]

**Review by rohith1122 — DISMISSED** [review]

File: test/demo/collection/demo_postman_collection.json (hunk 2 above)
Comment (mik3ola) [reply test/demo/collection/demo_postman_collection.json:1]: - Now updated to include correlation id key 'x_correlation_id' in the header on all scenarios.
- Readable format for the collection json fails git commit as githooks seems to have problems with the spacing/tabs and would only pass when collection json is alligned in a list format.

Comment (SLClayton) [reply test/demo/collection/demo_postman_collection.json:1]: Are there any other formatting scripts/online tools you could use that pass the checks. This isn't readable for review in a meaningful way.

**Review by mabe13 — APPROVED** [review]
LGTM 👍

**Review by mabe13 — APPROVED** [review]
LGTM 👍

**Review by SLClayton — APPROVED** [review]
These changes seem good to me.  It seems like it would be neater to store the body data as a nested json object. But I'm not familiar enough with this schema and have no reason to think storing them as raw strings would cause any issues either.

File: test/demo/collection/demo_postman_collection.json (hunk 2 above)
Comment (mik3ola) [reply test/demo/collection/demo_postman_collection.json]: Json format issue has now been resolved in latest commit.

**Pull request merged**